doc/rtd/development/first_PR.rst

```diff
@@ -0,0 +1,81 @@
+Submit your first pull request
```

Comment:
I decided to remove quite a lot from this file and instead point the GitHub quickstart guide. Let me know if you think that was a mistake.